labels/de-DE.json

```diff
@@ -136,6 +137,7 @@
   "customerAtCurbsideBlob4": "Notez s'il vous plaît...<br/><ul><li>Veuillez apporter votre pièce d'identité qui correspond au nom sur votre commande (sauf indication contraire).</li><li>Si votre colis n'est pas récupéré dans les 5 jours ouvrables, votre commande sera annulée et vous serez remboursé.</li><li>Lorsque vous arrivez au magasin, veuillez vous enregistrer via le lien fourni dans le prochain e-mail. À partir de là, vous pouvez fournir à l'associé du magasin les détails de la livraison de vos articles.</li></ul>",
   "customerAtCurbsideLetUsKnow": "LAISSEZ-NOUS SAVOIR LIVRER VOS ARTICLES!",
   "customerAtCurbsideWithYouShortly": "NOUS SERONS AVEC VOUS Bientôt!",
+  "customerCare": "Kundendienst",
```

Comment:
and this one is for?

Reply:
This as well came from the merged commit. It is not used for this ticket